src/core/Notification/Notification.md

```diff
@@ -0,0 +1,103 @@
+```js
+import {
+  Notification,
+  Button,
+  Paragraph
+} from 'suomifi-ui-components';
+
+<>
+  <Notification closeText="Close" smallScreen>
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification
+    status="error"
+    labelText="Lorem ipsum dolor sit"
+    smallScreen
+  >
+    Something went wrong. Please try again in a moment.
+  </Notification>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    headingVariant="h3"
+    actionElements={<Button variant="secondary">Button</Button>}
+  >
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    headingVariant="h2"
+    actionElements={
+      <>
+        <Button>Button</Button>
+        <Button variant="secondary">Button</Button>
+      </>
+    }
+  >
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification closeText="Close">
+    <Paragraph>
+      This is a long notification text. Lorem ipsum dolor sit amet,
+      consectetur adipiscing elit. Morbi est nulla, rhoncus ac odio
+      at, blandit ornare mi. Suspendisse potenti. Vestibulum ante
+      ipsum primis in faucibus orci luctus et ultrices posuere cubilia
+      curae; Suspendisse efficitur.
+    </Paragraph>
+    <Paragraph>
+      Et turpis eget tempor. Nam sed lacus ut lorem feugiat tincidunt.
+      Mauris quis nisi placerat, convallis turpis nec, auctor massa.
+      Nam ornare tortor id tortor tristique, in semper augue
+      fermentum. Nulla at pharetra leo, a ullamcorper metus. Donec
+      molestie velit tristique feugiat interdum. Donec ultrices
+      efficitur pellentesque. Sed luctus ac metus sed rhoncus.
+    </Paragraph>
+  </Notification>
```

Comment:
Maybe we could drop this other long version as it takes so much screen space and gives no additional value.

Current is about 90 words or so, so maybe 30-50 is already enough to show long stuff?

Reply:
I agree. I think only one with longer text is enought. All in all I'd try to cut the amount of examples here. We don't have to have every single combination of props portrayed here, but it's good to showcase the available variants and contents. Also, I don't think we can use the maintenance alert text here, since the notification is not meant for those kinds of messages.

Reply:
Fixed. Removed some of texts. Also removed Maintenance words.

Reply:
Now the lengths are better.

As @LJKaski raised it might be good idea to look together which examples were are about to show. Maybe there are too many examples to show at this time.

Reply:
Fixed. now there are 3 examples